app/components/progress-bar.hbs

```diff
@@ -0,0 +1,51 @@
+<div ...attributes class='progress-bar'>
+  <input
+    {{focus-when this.isEditable}}
+    aria-label='progress slider for IN-Progress tasks'
+    data-test-progress-bar
+    class='progress-bar__input-slider'
+    id='progress-bar-input'
+    type='range'
+    value={{@value}}
+    min='0'
+    max='100'
+    step='10'
+    {{on 'change' this.onChange}}
+    {{on 'input' this.onInput}}
+    disabled={{not this.isEditable}}
+  />
+  <button
+    data-test-progress-bar-button
+    {{on 'click' this.turnEditModeOn}}
+    class='progress-bar__button
+      {{unless this.isEditable "progress-bar__button--hover"}}'
+    type='button'
+  >
+    {{#if this.isEditable}}
+      <span class='progress-bar__button__text'>{{@value}}</span>
+    {{else}}
+      <svg
+        class='progress-bar__button__edit-icon'
+        viewBox='0 0 24 24'
+        fill='none'
+        xmlns='http://www.w3.org/2000/svg'
+      >
+        <path
+          class='progress-bar__button__edit-icon__inner'
+          opacity='0.5'
+          d='M4 20H8L18 10L14 6L4 16V20Z'
+          fill='#000000'
+        />
+        <path
+          class='progress-bar__button__edit-icon__outer'
+          d='M18 10L21 7L17 3L14 6M18 10L8 20H4V16L14 6M18 10L14 6'
+          stroke='#000000'
+          stroke-width='1.5'
+          stroke-linecap='round'
+          stroke-linejoin='round'
+        />
+      </svg>
```

Comment:
Please move svg to assets and use it from there

Reply:
I am dynamically changing the colour of the svg here. In this case would it be ok? Or is there any other way to do the same? 

<img width="576" alt="Screenshot 2023-07-25 at 11 59 31 PM" src="https://github.com/Real-Dev-Squad/website-my/assets/98796547/e4a292af-e3d6-48d2-8b95-9aa4ce95a991">